plugins/BlogLayout/views/shared/exhibit_layouts/double-image/form.php

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
I'd advocate for removing unnecessary blank lines in these.  Lines 1, 6, 11, 12 are unnecessary.  6 could be okay if you feel like you need to visually distinguish the two blocks of html here, but the others should definitely go.

Reply:
done